locale/id_ID/locale.xml

```diff
@@ -986,18 +1044,18 @@ Validasi mungkin gagal dalam pencarian URL memerlukan form data yang diposting.]
 	<message key="notification.mailListDescription">Masukkan alamat email Anda untuk menerima notifikasi langsung tentang konten baru yang penting dan ditambahkan ke jurnal.</message>
 	<message key="notification.notificationsPublicDescription">Halaman ini menunjukkan update penting yang berkaitan dengan jurnal ini seperti terbitan baru atau pengumuman. Anda bisa berlangganan ke notifikasi ini melalui RSS Feed (dengan meng-klik di gambar sebelah kanan), atau <![CDATA[  <a href="{$emailUrl}">melalui email.</a>]]></message>
 
-	<message key="notification.type.articleSubmitted">Artikel baru, "{$title}", telah diserahkan.</message>
+	<message key="notification.type.articleSubmitted">Artikel baru, "{$param}", telah diserahkan.</message>
```

Comment:
here and in the following lines, the parameter should be $title (was changes recently and apparantly after your commit, sorry...